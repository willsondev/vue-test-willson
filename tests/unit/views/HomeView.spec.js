import { shallowMount } from '@vue/test-utils';
import HomeView from '@/views/HomeView.vue';

describe('HomeView.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(HomeView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
