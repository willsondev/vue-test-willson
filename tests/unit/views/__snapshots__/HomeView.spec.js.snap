// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView.vue matches snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <hello-world-stub msg="Welcome to Your Vue.js App"></hello-world-stub>
</div>
`;
